Module/STIG/Functions.Checklist.ps1

```diff
@@ -398,26 +419,21 @@ function Get-FindingDetails
         $Setting
     )
 
-    switch ($setting.ResourceID)
+    switch ($setting.ResourceID) 
```

Comment:
Trailing whitespace.

Reply:
Style guidelines suggest [each curly brace should be preceded by a newline](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#one-newline-before-braces)

Could you correct this here down to line 436

Also remove the whtiespace inside the conditions.